### PR TITLE
feat(cargo): add cargo plugins

### PR DESCRIPTION
**Pull request by grant0417**
Add `udeps`, `deny`, `bloat`, `sort`, and `fuzz`

### CODE REVIEW

File: src/cargo.ts [hunk 1]
```diff
@@ -5667,6 +5679,825 @@ const completionSpec: (toolchain?: boolean) => Fig.Spec = (
...
+            description: "Package(s) to check",
+            args: {
+              name: "SPEC",
+            },
+          },
+          {
+            name: "--all",
+            description: "Alias for --workspace (deprecated)",
+            hidden: true,
+          },
```
Comment (fedeci) [inline src/cargo.ts:5703]:
```suggestion
          {
            name: "--all",
            description: "Alias for --workspace (deprecated)",
            hidden: true,
            deprecated: true,
          },
```

File: src/cargo.ts [hunk 2]
```diff
@@ -5667,6 +5679,825 @@ const completionSpec: (toolchain?: boolean) => Fig.Spec = (
...
+              {
+                name: "init",
+                description: "Creates a cargo-deny config from a template",
+              },
+              {
+                name: "list",
+                description:
+                  "Outputs a listing of all licenses and the crates that use them",
+              },
+            ],
```
Comment (fedeci) [inline src/cargo.ts:6047]:
```suggestion
            args: {
              template: "help",
              isOptional: true,
            },
```

**Review by fedeci — COMMENTED** [review]
@grant0417 just two comments to solve then it lgtm! Feel free to merge this in when it is ready.

**Review by SeparateRecords — APPROVED** [review]
lgtm!

**Pull request merged**